pkg/driver/nri_hooks.go

```diff
@@ -255,20 +271,27 @@ func (np *NetworkDriver) StopPodSandbox(ctx context.Context, pod *api.PodSandbox
 	for deviceName, config := range podConfig {
 		if err := nsDetachNetdev(ns, config.NetworkInterfaceConfigInPod.Interface.Name, config.NetworkInterfaceConfigInHost.Interface.Name); err != nil {
 			klog.Infof("fail to return network device %s : %v", deviceName, err)
+			nriPluginRequestsTotal.WithLabelValues(methodStopPodSandbox, statusFailed).Inc()
 		}
 
 		if !np.rdmaSharedMode && config.RDMADevice.LinkDev != "" {
 			if err := nsDetachRdmadev(ns, config.RDMADevice.LinkDev); err != nil {
 				klog.Infof("fail to return rdma device %s : %v", deviceName, err)
+				nriPluginRequestsTotal.WithLabelValues(methodStopPodSandbox, statusFailed).Inc()
 			}
 		}
 	}
+	nriPluginRequestsTotal.WithLabelValues(methodStopPodSandbox, statusSuccess).Inc()
 	return nil
 }
 
 func (np *NetworkDriver) RemovePodSandbox(_ context.Context, pod *api.PodSandbox) error {
 	klog.V(2).Infof("RemovePodSandbox Pod %s/%s UID %s", pod.Namespace, pod.Name, pod.Uid)
+	start := time.Now()
+	defer func() { nriPluginRequestsLatencySeconds.WithLabelValues(methodRemovePodSandbox).Observe(time.Since(start).Seconds()) }()
```

Comment:
I prefer to have multiple lines, it reads better

Reply:
Got it, updated.